src/app/wallets/send-on-chain.ts

```diff
@@ -278,15 +278,15 @@ const executePaymentViaIntraledger = async <
     const converter = NewDisplayCurrencyConverter(displayCentsPerSat)
 
     let metadata:
-      | NewAddOnChainIntraledgerSendLedgerMetadata
-      | NewAddOnChainTradeIntraAccountLedgerMetadata
+      | AddOnChainIntraledgerSendLedgerMetadata
+      | AddOnChainTradeIntraAccountLedgerMetadata
     let additionalDebitMetadata: { [key: string]: string | undefined } = {}
     if (senderWallet.accountId === recipientWallet.accountId) {
       ;({ metadata, debitAccountAdditionalMetadata: additionalDebitMetadata } =
         LedgerFacade.OnChainTradeIntraAccountLedgerMetadata({
           payeeAddresses,
           sendAll,
-          paymentFlow,
+          paymentAmounts: paymentFlow,
```

Comment:
what is there a difference between paymentFlow and paymentAmounts? or renaming is a big change?

Reply:
`paymentAmounts` is a subset of `paymentFlow`. The hack before this was to cast the param type to get the 4 properties needed. Now we've explicitly declared a new subset type to avoid the cast. Since it's a subset of `paymentFlow`, I believe we can pass the whole object here instead of selecting the 4 properties from it that we actually need

... or?